build.sbt

```diff
@@ -38,4 +37,86 @@ excludeDependencies ++= Seq(
 
 assemblyOutputPath in assembly := file("/tmp/conseil.jar")
 
-scalacOptions ++= ScalacOptions.common
\ No newline at end of file
+scalacOptions ++= ScalacOptions.common
+
+//uses git tags to generate the project version
+//see https://github.com/sbt/sbt-git#versioning-with-git
+enablePlugins(GitVersioning)
+
+/* The versioning scheme is
+ *  - use a major number as the platform version
+ *  - add a date reference in form of yyww (year + week in year)
+ *  - use the latest git tag formatted as "ci-release-<xyz>" and take the numbers from xyz, increasing it
+ * Compose the three separated by "dots" to have the version that will be released
+ * The Git plugin will add a trailing "-SNAPSHOT" if there are locally uncommitted changes
+ */
+val majorVersion = 0
```

Comment:
This will have the meaning of a "platform number" as in `Conseil-1`, `Conseil-2`